### PR TITLE
Forgo all checks when running __sdk_completion

### LINKED ISSUE #912: Completion script initialisation problems
Using sdkman SDKMAN 5.11.2+698 on Ubuntu Linux 20.04 and bash 5.0.17(1) with the necessary code in .bashrc to put it in the path.

When I start a new terminal, i.e. invoke a new bash, output like the following *sometimes* appears on the console:

```
ommand not found
36m====: command not found

Command '0music' not found, did you mean:

  command 'music' from deb music-bin (1.1.16-1.1build2)

Try: sudo apt install <deb name>


Command '0music' not found, did you mean:

  command 'music' from deb music-bin (1.1.16-1.1build2)

Try: sudo apt install <deb name>


Command '0music' not found, did you mean:

  command 'music' from deb music-bin (1.1.16-1.1build2)

Try: sudo apt install <deb name>

================================================================================: command not found
```

I did not manage to find out why/when this happens, it appears to be almost random, though it usually happens when opening the *first* terminal after a restart or after a longer time. 

I have observed this behaviour for several weeks now and I am certain this only happens when the SDKMAN-related code is active in .bashrc, never happens when I comment that out. 

The "command not found"  complaints I get appear to come from strings inside the bash prompt definition and from the names of random directories in my home directory. I have no idea at all what could be going on here, but it became so annoying that I usually deactivate sdkman completely. 

So sadly, I cannot provide steps to reproduce because this problem occurs seemingly randomly (but often enough to be very annoying).

### PR DESCRIPTION
**Pull request by helpermethod**
Fixes #912.

### CODE REVIEW

File: src/main/bash/sdkman-main.sh [hunk 1]
```diff
@@ -29,6 +29,7 @@ function sdk() {
 		COMMAND="list"
 		;;
 	v)
+		COMMAND="version"
```
Comment (marc0der) [inline src/main/bash/sdkman-main.sh:32]: Do we need this twice?

Comment (helpermethod) [reply src/main/bash/sdkman-main.sh:32]: Ah sorry, I've messed up.

Comment (marc0der) [reply src/main/bash/sdkman-main.sh]: No worries! 😄

**Comment by helpermethod** [comment]
Removed the unnecessary code change.

**Review by marc0der — APPROVED** [review]

**Comment by marc0der** [comment]
LGTM!

**Pull request merged**